cypress/utils/dashboards/commands.js

```diff
@@ -45,123 +45,3 @@ Cypress.Commands.add(
-  // Creates an index pattern within the workspace using cluster
-  // Don't use * in the indexPattern it adds it by default at the end of name
-  'createWorkspaceIndexPatterns',
-  (opts) => {
-    const {
-      url,
-      workspaceName,
-      indexPattern,
-      timefieldName,
-      indexPatternHasTimefield = true,
-      dataSource,
-      isEnhancement = false,
-    } = opts;
-
-    // Navigate to Workspace Specific IndexPattern Page
-    cy.navigateToWorkSpaceSpecificPage({
-      url,
-      workspaceName,
-      page: 'indexPatterns',
-      isEnhancement,
-    });
-    cy.getElementByTestId('createIndexPatternButton').click();
-    cy.waitForLoader(isEnhancement);
-
-    if (dataSource) {
-      // First select "Use external data source connection" radio button
-      // Ensure the radio is enabled and need to force click it
-      // This is due to data-test-subj="createIndexPatternStepDataSourceUseDataSourceRadio") is on the parent div, not on the actual radio input element
-      cy.get('input#useDataSource').should('not.be.disabled').click({ force: true });
-
-      cy.get('[type="data-source"]').contains(dataSource).click();
-    }
-
-    cy.getElementByTestId('createIndexPatternStepDataSourceNextStepButton').click();
-
-    cy.getElementByTestId('createIndexPatternNameInput')
-      .should('be.visible')
-      .clear()
-      .type(indexPattern);
-    cy.getElementByTestId('createIndexPatternGoToStep2Button').click();
-
-    // wait for the select input if it exists
-    if (indexPatternHasTimefield || timefieldName) {
-      cy.getElementByTestId('createIndexPatternTimeFieldSelect').should('be.visible');
-    }
-
-    if (indexPatternHasTimefield && !!timefieldName) {
-      cy.getElementByTestId('createIndexPatternTimeFieldSelect').select(timefieldName);
-    } else if (indexPatternHasTimefield && !timefieldName) {
-      cy.getElementByTestId('createIndexPatternTimeFieldSelect').select(
-        "I don't want to use the time filter"
-      );
-    }
-
-    cy.getElementByTestId('createIndexPatternButton').should('be.visible').click();
-    cy.getElementByTestId('headerApplicationTitle').contains(indexPattern);
-  }
-);
-
-Cypress.Commands.add(
-  // deletes an index pattern within the workspace
-  // Don't use * in the indexPattern it adds it by default at the end of name
-  'deleteWorkspaceIndexPatterns',
-  (opts) => {
-    const { url, workspaceName, indexPattern, isEnhancement = false } = opts;
-
-    // Navigate to Workspace Specific IndexPattern Page
-    cy.navigateToWorkSpaceSpecificPage({
-      url,
-      workspaceName,
-      page: 'indexPatterns',
-      isEnhancement,
-    });
-
-    cy.contains('a', indexPattern).click();
-    cy.getElementByTestId('deleteIndexPatternButton').should('be.visible').click();
-    cy.getElementByTestId('confirmModalConfirmButton').should('be.visible').click();
-
-    // wait until delete is done
-    cy.getElementByTestId('headerApplicationTitle').should('be.visible');
-  }
-);
```

Comment:
the associated `index.d.ts` can be deleted

Reply:
good call. will update.